src/data/items.ts

```diff
@@ -2,20 +2,105 @@ import { CREATE, DELETE, GET, UPDATE } from "."
 import { start, stop } from "../components/progress"
 import { throwError } from "../error"
 import { writable } from "svelte/store"
+import type { Claim } from "./claims"
+
+export type ItemCoverageStatus = 'Draft' | 'Pending' | 'Approved' | 'Denied';
+export type PolicyType = 'Household' | 'Corporate';
+
+export type RiskCategory = {
+  created_at: string /*Date*/;
+  id: {
+    UUID: any;
+    Valid: boolean;
+  };
```

Comment:
This seems odd to me... let's verify with the API team that a RiskCategory's `id` is an array, rather than just a number or string.

Reply:
Looks like that was a mistake in the API. This can be changed to
```suggestion
  id: string;
```